### DISCUSSION TITLE
updated hostname broke certificates

### DISCUSSION THREAD

**Discussion by wtl06639** [Security-Onion-Solutions/securityonion#6759] [Unsupported Versions] [answered]
I had to change the hostname of my manager and sensors. This broke all the SSL certificates for each of the difference services. Any recommendations on the easiest way to resolve this? Not sure if just doing a fresh install would be quicker or recreating the certs. Not sure if that would still allow it to update smoothly again or not without throwing out the cert name issue.

**Comment by TOoSmOotH** [comment, marked as answer]
You are better off re-installing vs trying to make this work. There are too many things you would need to manually change to remedy the problem and even then there is no guarantee that it will work properly.
